tests/test_rpm_verifier.py

```diff
@@ -15,178 +15,164 @@
     ProcessedImage,
     generate_output,
     get_rpmdb,
+    get_rpms_data,
     get_unsigned_rpms,
-    parse_image_input,
 )
 
 
+def test_get_rpmdb(tmp_path: Path) -> None:
+    """Test get_rpmdb"""
+    image = "my-image"
+    mock_runner = MagicMock()
+    out = get_rpmdb(
+        container_image=image,
+        target_dir=tmp_path,
+        runner=mock_runner,
+    )
+    assert mock_runner.call_count == 1
+    assert out == tmp_path
+
+
 @pytest.mark.parametrize(
-    ("processed_images", "expected_print", "expected_failures_exist"),
+    "test_input,expected",
```

Comment:
I'd prefer we keep the format of providing the list of parameters uniform throughout the file (e.g. use a tuple everywhere), but if we use a string, let's add spaces after the commas:
```suggestion
    "test_input, expected",
```

Reply:
Changed to tuples